features/GEM003_Unique-representation-identifier.feature

```diff
@@ -0,0 +1,11 @@
+@informal-proposition
+@GEM
+Feature: GEM003 - Unique Representation Identifer
+
+  Scenario: No Shape Representation identifier shall be used twice within the product representation of an element 
+
+      Given An IfcProductDefinitionShape
```

Comment:
It might make sense to start at the IfcProduct, which has attribute Representations of type IfcProductDefinitionShape. The Product (wall, column, ...) is much more recognizable to the end user.